package.json

```diff
@@ -104,6 +104,10 @@
         "description": "Set to `true` to get index.pdf generated next to your index.html",
         "default": false
       },
+      "favicon": {
+        "description": "URL of the favicon",
+        "default": "https://www.asyncapi.com/favicon.ico"
```

Comment:
Could we also add possibility to pass favicon from local disk? We should check if given path is local/absolute in filesystem and then copy it to the output folder. Do you wanna handle that?

Reply:
I think the option definitely makes sense. I was not sure though, how the copying would exactly work. Also the sinlge file option would have to handled differently, maybe by encoding the file as base64. I can look into that though.

Reply:
You can copy something to the final output using `generate:after` hook like here https://github.com/asyncapi/html-template/blob/master/hooks/02_renameOutFile.js in mentioned file you can see how we deal with targetDir (output path) and passed parameters (templateParams). As for `singleFile` feature - yeah we should try to convert favicon to the base64 and use `data:image/png;base64,{icon}` format. I can make it for you if you don't have time :)